docs/development/extensions-core/datasketches-tuple.md

```diff
@@ -207,3 +207,39 @@ Returns a human-readable summary of a given ArrayOfDoublesSketch. This is a stri
   "field"  : <post aggregator that refers to an ArrayOfDoublesSketch (fieldAccess or another post aggregator)>
 }
 ```
+
+
+### Constant ArrayOfDoublesSketch 
+
+You can use the this post aggregator to add a Base64-encoded constant ArrayOfDoublesSketch value for use in other post-aggregators.
```

Comment:
```suggestion
This post aggregator adds a Base64-encoded constant ArrayOfDoublesSketch value that you can use in other post aggregators.
```
Not sure if what is meant by "adds". Would it be accurate to say "generates" or "outputs"?